### PR TITLE
docs: update release doc

### PR DESCRIPTION
**Pull request by xiaobiaozhao**

### CODE REVIEW

File: community/create-a-release.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ svn ci -m "add gpg key for YOUR_NAME" # Later on, if you are asked to enter a us
 ## Create source releases and stage
 
 1. Checkout to the RELEASE BRANCH and cherry-pick commits to release
-2. Use `./x.py package source --version ${release_version} -rc ${candidate_number}` to create release tarball
+2. Use `./x.py package source --v ${release_version} -rc ${candidate_number}` to create release tarball
```
Comment (git-hulk) [inline community/create-a-release.md:173]:
```suggestion
2. Use `./x.py package source -v ${release_version} -rc ${candidate_number}` to create release tarball
```

Comment (xiaobiaozhao) [reply community/create-a-release.md]: fixed

**Review by git-hulk — APPROVED** [review]

File: community/create-a-release.md [hunk 2]
```diff
@@ -212,9 +212,10 @@ Docker and Docker Buildx is required for this step. Docker Buildx should be bund
 
 1. `docker login` to your own DockerHub account.
 2. Build and push docker images to your personal repository on DockerHub:
+3. MORE BUILD ARGS can be adjusted according to the performance of your computer
 
 ```shell
-docker buildx build --platform linux/amd64,linux/arm64 --tag ${your_dockerhub_username}/kvrocks:${release_version} --tag ${your_dockerhub_username}/kvrocks:latest . --output-type=registry
+docker buildx build --platform linux/amd64,linux/arm64 --tag ${your_dockerhub_username}/kvrocks:${release_version} --tag ${your_dockerhub_username}/kvrocks:latest . --output "type=registry" --build-arg MORE_BUILD_ARGS=-j12
```
Comment (tisonkun) [inline community/create-a-release.md:218]: Generally, we don't need to specify `-j12` but let the platform to determinate parallelism.

Comment (PragmaTwice) [reply community/create-a-release.md:218]: What is the meaning of `12` here? Why is `12` fit for any CPU settings? We do not need these nonsense numbers.

**Pull request merged**